ssh-restrict

```diff
@@ -56,6 +56,7 @@ except KeyError:
     sys.exit(RETURN_CODE_WRONG_USAGE)
 
 config = ConfigParser.RawConfigParser()
+config.optionxform = lambda option: option
```

Comment:
Please add a comment here why this is necessary

Reply:
Added comment: 
`# Override optionxform to bypass conversion of keys to lowercase`
I assume that explains the circumstance for the change. 
cheers, 
--urs